Update index.md (WebTransport API) (#29745)
mdn/content: files/en-us/web/api/webtransport_api/index.md

These include:

> - Head-of-line blocking

Reviewer: Do you think it would be worth adding strong formatting to the `<dt>` elements? I think it would be more scannable that way

Author: Yup 👍🏽

> - Head-of-line blocking

Reviewer 2: ```suggestion
- **Head-of-line blocking**
```

  - : HTTP/2 allows multiplexing, so a single connection can stream multiple resources simultaneously. However, if a single resource fails, all other resources on that connection are held up until any missing packets are retransmitted. With QUIC, only the failing resource is affected.

> - Faster performance

Reviewer 2: ```suggestion
- **Faster performance**
```

>   - : QUIC is more performant than TCP in many ways. QUIC can handle security features itself, rather than handing responsibility off to other protocols like TLS, meaning fewer round trips. And streams provide better transport efficiency than the older packet mechanism. This can make a significant difference, especially on high-latency networks.

Reviewer 2: ```suggestion
  - : QUIC is more performant than TCP in many ways. QUIC can handle security features by itself, rather than handing responsibility off to other protocols like TLS — meaning fewer round trips. And streams provide better transport efficiency than the older packet mechanism. That can make a significant difference, especially on high-latency networks.
```

> - Better network transitions

Reviewer 2: ```suggestion
- **Better network transitions**
```

>   - : QUIC uses a unique connection ID to handle the source and destination of each request to ensure that packets are delivered correctly. This ID can persist between different networks, meaning that for example a download can continue interrupted if you switch from Wifi to a mobile network. HTTP/2 on the other hand uses IP addresses as identifiers, so network transitions can be problematic.

Reviewer 2: ```suggestion
  - : QUIC uses a unique connection ID to handle the source and destination of each request — to ensure that packets are delivered correctly. This ID can persist between different networks, meaning that, for example, a download can continue interrupted if you switch from Wifi to a mobile network. HTTP/2, on the other hand, uses IP addresses as identifiers, so network transitions can be problematic.
```

> - Unreliable transport

Reviewer 2: ```suggestion
- **Unreliable transport**
```

### Reliable transmission via streams

> "Reliable" means that transmission and order of data are guaranteed. This provides slower delivery (albeit faster than with WebSockets), and is needed in situations where reliability and ordering are important (like chat applications, for example).

Reviewer 2: ```suggestion
"Reliable" means that transmission and order of data are guaranteed. That provides slower delivery (albeit faster than with WebSockets), and is needed in situations where reliability and ordering are important (like chat applications, for example).
```

> "Reliable" means that transmission and order of data are guaranteed. That provides slower delivery (albeit faster than with WebSockets), and is needed in situations where reliability and ordering are important (like chat applications, for example).

Reviewer 2: ```suggestion
"Reliable" means that transmission and order of data are guaranteed. That provides slower delivery (albeit faster than with WebSockets), and is needed in situations where reliability and ordering are important (such as chat applications, for example).
```
